src/promptflow/promptflow/executor/flow_executor.py

```diff
@@ -936,12 +936,14 @@ def _exec(
             # KeyboardInterrupt will be raised after asyncio finishes its signal handling
             # End run with the KeyboardInterrupt exception, so that its status will be Canceled
             flow_logger.info("Received KeyboardInterrupt, cancel the run.")
+            # Update the run info of those running nodes to a canceled status.
```

Comment:
Will a conflict occur here? An async function would also be cancelled in it, right?

Reply:
An async function will be cancelled before this part.